Sources/SwiftParser/Statements.swift

```diff
@@ -219,15 +219,15 @@ extension Parser {
   ///     optional-binding-condition → 'let' pattern initializer? | 'var' pattern initializer? |
   ///                                  'inout' pattern initializer?
   @_spi(RawSyntax)
-  public mutating func parseConditionElement() -> RawConditionElementSyntax.Condition {
+  public mutating func parseConditionElement(lastBindingKind: RawTokenSyntax?) -> RawConditionElementSyntax.Condition {
```

Comment:
Could you add a doc comment describing what `lastBindingKind` does?

Reply:
Something like this?

Reply:
Looks good 👍🏽